src/js/tech/tech.js

```diff
@@ -234,6 +234,13 @@ class Tech extends Component {
   }
 
   /**
+   * Reset the tech. Removes all sources and resets readyState.
+   *
+   * @method reset
+   */
+  reset() {}
```

Comment:
How will the Flash tech react to this? It seems like reset() will only work for HTML5.

Related: I forgot that src() accepts an array of source objects. What do you think about making `[]` equivalent to a reset?